### DISCUSSION TITLE
Release v1.1.3 / 5.56.3

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#1976] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

### -  OpenKeyPath settings do not work between v1.1.1 and v1.1.3: https://github.com/sandboxie-plus/Sandboxie/issues/2006 https://github.com/sandboxie-plus/Sandboxie/issues/1276#issuecomment-1179549904

This release changes how sandboxie marks deleted host files as deleted, instead of creating dummy entries it uses an internal data structure to keep the stat of changes to the host file system. With the new mechanics it is possible to rename folders and keys properly.
This new behavior needs to be enabled by specifying UseFileDeleteV2=y sand UseRegDeleteV2=y in later builds this will be enabled by default, hence please test this build and report back if there are any issues.

This build also adds a new UI functionality, sandman.exe now monitors the size of sandboxes and reports it in the info column, this functionality needs to be enabled in the global settings.

Last but not least this build reduces the cpu usage of sandman.exe a lot.

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

# ChangeLog

### Added
- added group first sorting [#1922](https://github.com/sandboxie-plus/Sandboxie/issues/1922)

### Changed
- updated Classic UI Swedish translation (thanks pb1)
- restored Plus UI Turkish translation [#1419](https://github.com/sandboxie-plus/Sandboxie/issues/1419) (thanks fmbxnary)

### Fixed
- fixed issue with recovery window on delete [#1948](https://github.com/sandboxie-plus/Sandboxie/issues/1948)
- fixed double-click issue on path column [#1951](https://github.com/sandboxie-plus/Sandboxie/issues/1951)
- "AllowBoxedJobs=n" is back to the default behaviour as issues were reported [#1954](https://github.com/sandboxie-plus/Sandboxie/issues/1954)
- fixed issue with internet block [#1955](https://github.com/sandboxie-plus/Sandboxie/issues/1955)
- fixed grouping issue in the Plus UI [#1950](https://github.com/sandboxie-plus/Sandboxie/issues/1950)
- fixed issue with CredentialUIBroker.exe on win 11 with win32k hooks [#1839](https://github.com/sandboxie-plus/Sandboxie/issues/1839)
- fixed issue with delete V2 [#1939](https://github.com/sandboxie-plus/Sandboxie/issues/1939)

### Added
- added missing file recovery log to SandMan [#425](https://github.com/sandboxie-plus/Sandboxie/issues/425)
- the immediate recovery window will now auto-close when all files have been recovered [#1498](https://github.com/sandboxie-plus/Sandboxie/issues/1498)
- the immediate recovery window of SandMan is always on top by default like in SbieCtrl; this can be disabled with "Options/RecoveryOnTop=n" [#1465](https://github.com/sandboxie-plus/Sandboxie/issues/1465)
- added option to toggle immediate recovery from the presets submenu [#1653](https://github.com/sandboxie-plus/Sandboxie/issues/1653)
- added option to disable file recovery and message pop-up globally
- added per box refresh option [#1945](https://github.com/sandboxie-plus/Sandboxie/issues/1945)

### Changed
- the desktop security workaround used for Chrome, Firefox and Acrobat can now be enabled for all processes using "UseSbieDeskHack=y"
- improved double-click behaviour [#1935](https://github.com/sandboxie-plus/Sandboxie/issues/1935)
- box size info is refreshed on file recovery

### Fixed
- fixed issue with unnecessary Sandboxie config reloads introduced in 1.1.1 [#1938](https://github.com/sandboxie-plus/Sandboxie/issues/1938)
- fixed issue with recovery window focus [#1374](https://github.com/sandboxie-plus/Sandboxie/issues/1374)
- fixed issues with desktop objects introduced in 1.1.1 [#1934](https://github.com/sandboxie-plus/Sandboxie/issues/1934)
- fixed issues with Edge startup boost using a GPO preset [#1913](https://github.com/sandboxie-plus/Sandboxie/issues/1913)

### Added
- compatibility templates can now be viewed from the settings window [#1891](https://github.com/sandboxie-plus/Sandboxie/issues/1891)
- the refresh command is now bound to F5 [#1885](https://github.com/sandboxie-plus/Sandboxie/issues/1885)
- added more first start wizard options
- added option to permanently disable immediate recovery for any given box when it opens [#1478](https://github.com/sandboxie-plus/Sandboxie/issues/1478)
- double click on the path column now opens the box root in explorer [#1924](https://github.com/sandboxie-plus/Sandboxie/issues/1924)

### Changed
- changed Move Box behaviour [#1879](https://github.com/sandboxie-plus/Sandboxie/issues/1879)
- improved implementation of the PreferExternalManifest option
- Win32k hooks are now by default only used for Edge and Chromium apps as they cause issues with other software [#1902](https://github.com/sandboxie-plus/Sandboxie/issues/1902) [#1912](https://github.com/sandboxie-plus/Sandboxie/issues/1912) [#1897](https://github.com/sandboxie-plus/Sandboxie/issues/1897)
- "AllowBoxedJobs=y" is now the default behaviour

### Fixed
- fixed Edge issue with Windows 11 after KB5014019
- fixed issues with the new delete V2 mechanism when using "SeparateUserFolders=y" [#1885](https://github.com/sandboxie-plus/Sandboxie/issues/1885)
- fixed credential issue [#1770](https://github.com/sandboxie-plus/Sandboxie/pull/1770)
- fixed force process priorities [#1883](https://github.com/sandboxie-plus/Sandboxie/issues/1883)
- fixed issues with the new delete V2 mechanism
- fixed issue with the Windows 11 menu on older Windows builds [1877](https://github.com/sandboxie-plus/Sandboxie/issues/1877)
- refresh now works without WatchBoxSize option [#1885](https://github.com/sandboxie-plus/Sandboxie/issues/1885)
- fixed crash issue with WatchBoxSize=true [#1885](https://github.com/sandboxie-plus/Sandboxie/issues/1885)
- fixed issue with recovery folder paths [#1840](https://github.com/sandboxie-plus/Sandboxie/issues/1840)
- fixed issues with Sbie desktop and wndStation affecting Acrobat Reader [#1863](https://github.com/sandboxie-plus/Sandboxie/issues/1863)
- fixed issues with box grouping [#1921](https://github.com/sandboxie-plus/Sandboxie/issues/1921) [#1920](https://github.com/sandboxie-plus/Sandboxie/issues/1920)
- fixed issues when changing language [#1914](https://github.com/sandboxie-plus/Sandboxie/issues/1914)
- fixed issue with BreakoutFolder [#1908](https://github.com/sandboxie-plus/Sandboxie/issues/1908)
- fixed issue with SbieDll.dll for x86 exception handling
- fixed issues with application-specific hives (RegLoadAppKey) affecting Visual Studio [#1576](https://github.com/sandboxie-plus/Sandboxie/issues/1576) [#1452](https://github.com/sandboxie-plus/Sandboxie/issues/1452)

### Added
- added support for NtRenameKey (this requires UseRegDeleteV2=y) [#205](https://github.com/sandboxie-plus/Sandboxie/issues/205)
- added box size info [#1780](https://github.com/sandboxie-plus/Sandboxie/issues/1780)

### Changed
- reworked the mechanism Sandboxie uses to mark host files as deleted
-- under the new behaviour a data file (FilePaths.dat) is created in the box root instead of dummy files
-- it can be enabled with UseFileDeleteV2=y and also for the registry with UseRegDeleteV2=y which creates a reg file (RegPaths.dat)
- reworked the TlsNameBuffer mechanism to be more versatile and less error-prone
- significantly reduced the CPU usage of SandMan.exe

### Fixed
- fixed folder rename issues (this requires UseFileDeleteV2=y) [#71](https://github.com/sandboxie-plus/Sandboxie/issues/71)
- fixed issue with process access [#1603](https://github.com/sandboxie-plus/Sandboxie/issues/1603)
- fixed translation issue [#1864](https://github.com/sandboxie-plus/Sandboxie/issues/1864)
- fixed UI issue with the box selection window [#1867](https://github.com/sandboxie-plus/Sandboxie/issues/1867)
- fixed UI issue when switching languages [#1871](https://github.com/sandboxie-plus/Sandboxie/issues/1871)

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.1.3'>Release v1.1.3 / 5.56.3</a>.</em>

**Comment by SiNONiMiTY** [comment]
Hi @DavidXanatos, this release contains the changelog for the previous releases, making it redundant and confusing. Please help edit the release info.

**Reply by DavidXanatos** [reply]
that is intentional as its no longer a pre release as uyers may skip pre release builds i thought it would be a good idea to always have the changelog since the last non pre release build included

**Reply by SiNONiMiTY** [reply]
I see, thanks for the clarification.

**Comment by fmbxnary** [comment]
hi @DavidXanatos Turkish translation has just been restored and Turkish does not appear on the installation screen. Is this normal?

**Reply by isaak654** [reply]
I believe it's the issue #1556

**Comment by DavidXanatos** [comment]
No, wil have to check that out perhaps i forgot to update the pri file

Am Di., 21. Juni 2022 um 16:24 Uhr schrieb Fatih Mehmet Baydar <
***@***.***>:

**Comment by BilleBarrett** [comment]
Every morning when I start up Chrome I get these 2 errors
CompPkgSrv.exe (4560): SBIE2206 Failed processing AutoExec setting -2 [C0000022]
  CompPkgSrv.exe (4560): SBIE2312 Could not enable BrowseNewProcess setting:  [77 / C0000001]
I dismiss them and use Chrome
Version 103.0.5060.53 (Official Build) (64-bit)
When I check the current version I get this error
GoogleUpdate.exe (10648): SBIE2224 Sandboxed program has crashed: GoogleUpdate.exe [DefaultBox]
Do I have a setting wrong
Thank you

**Comment by Githubnaire** [comment]
Hi @DavidXanatos, I am concerned about a very important problem that I have encountered, while using Sandboxie Plus. How do I fix the problem with the camera, which is that any application that uses the camera recognizes it, but there is simply no image. By the example of Zoom, instead of a camera image, just a black image. Or OBS, for example, which also doesn't have a working camera. If you run it outside the sandbox, the camera works without problems. Can you solve this problem with camera. Because of this problem I can not use it during a video call. I have tried all the settings but nothing works.

My data:

Windows 11 Home 
Version: 21H2
OS Build: 22000.778
64-bit operating system, x64-based processor
Release: SandBoxie-Plus 1.1.3

Thank you in advance!

**Reply by kntjspr** [reply]
Did you allow the sandbox to access hardware? You can locate in on Sandbox options -> Advanced Options -> Isolation
![image](https://user-images.githubusercontent.com/65537922/178420626-2600c16f-a3a5-4fbd-b445-9d00f79c12cc.png)
Not sure if this helps tho

**Comment by BilleBarrett** [comment]
Just for reference, I have Sandboxie-Plus running on a Server 2016 and if you try to run outside of the Sandbox it does not work
everything else work great

**Comment by DavidXanatos** [comment]
for me it works:
[image: grafik.png]


Am Di., 21. Juni 2022 um 18:46 Uhr schrieb David Xanatos <
***@***.***>:

**Reply by fmbxnary** [reply]
Yes it's been working fine for a while.
